docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,211 @@
+
+4. To start the cluster, use either hz-start or hz-start.bat depending on your operating system. Once you run the start script, you should see the Hazelcast logs in the terminal.
+
+== Installing and Starting with the Client
+
+Create a new folder by
+
+[source, bash]
+mkdir hazelcast-python-client-getting-started
+
+Navigate to that folder by
+
+[source, bash]
+cd hazelcast-python-client-getting-started
+
+Hazelcast Python Client can be installed by pip3. Run the following command.
+
+[source, bash]
+pip3 install hazelcast-python-client
+
+Create a file named tutorial.py
+
+Put the following code inside of tutorial.py.
+
+[source, python]
+----
+import hazelcast
+
+client = hazelcast.HazelcastClient()
+
+client.shutdown()
+----
+To run this Python script, use the following command in the terminal.
+
+[source, bash]
+python3 tutorial.py
+
+First line imports the Hazelcast Python client.
+
+Following line creates and starts a new Hazelcast Client. To configure the client, you can check https://hazelcast.readthedocs.io/en/stable/client.html#hazelcast.client.HazelcastClient[API documentation].
+
+== Use Map
+
+A Hazelcast map is a distributed key-value store, similar to Python Map structure. In the following example, we will work with map entries where the keys are session ids and the values are emails.
+
+[source, python]
+----
+import hazelcast
+
+client = hazelcast.HazelcastClient()
+
+some_map = client.get_map("some-map")
+
+some_map.set("12345", "example1@email.com")
+some_map.set("12346", "example2@email.com")
+
+entries = some_map.entry_set().result()
+
+for key, value in entries:
+    print("%s -> %s" % (key, value))
+
+client.shutdown()
+----
+
+
+The output snippet is
+[source, bash]
+----
+example1@email.com
+example2@email.com
+----
+
+Since Hazelcast clients work asynchronous, our output is not in order.
+
+Following line returns a map proxy object for the 'some-map' named map.
+
+[source, python]
+some_map = client.get_map("some-map")
+
+If there is not a map named ”some-map”, it will be automatically created. All the clients connected to the same cluster will have access to same map.
+
+With these lines, client adds data to our “some-map”. The first parameter is the key of the entry, the second one is the value as mentioned above.
+
+[source, python]
+----
+some_map.set("12345", "example1@email.com")
+some_map.set("12346", "example2@email.com")
+----
+
+Then, we get the the values with the entry_set().result() method. You may wonder why we have used result()method over the entry_set(). This is because the Hazelcast Python client is designed to be fully asynchronous. Every method call over distributed objects such as put(), get(), entry_set(), etc. will return a https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Future[Future] object that is similar to the Future class of the https://docs.python.org/3/library/concurrent.futures.html#module-concurrent.futures[concurrent.futures] module.
```

Comment:
Let's remove the reference to the Python3 `Future`. The `Future` in the Python client is different from the Python3 one, although they have the similar concept.

Reply:
I have added it while searching docs. Here is what I referenced [link](https://hazelcast.readthedocs.io/en/stable/getting_started.html#:~:text=You%20may%20wonder,concurrent.futures%20module.)